src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, 'font-family: Roboto, Arial, sans-serif;' could be 'font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;' to provide more robust font coverage.